docs/api/security/Devicekey.md

```diff
@@ -19,14 +19,18 @@ The characteristics required by this root of trust are:
 
 The DeviceKey feature keeps the root of trust key in internal storage, using the NVStore component. Internal storage provides protection from external physical attacks to the device.
 
-The root of trust is generated at the first use of DeviceKey if the true random number generator is available in the device. If no true random number generator is available, you must pass the injected root of trust key to the DeviceKey before you call the key derivation API.
+The root of trust is not generated at the first use of DeviceKey. It must be created explicitly before. If the true random number generator is available in the device it is used for root of trust generation. Otherwise injected root of trust must be passed to the DeviceKey before key derivation API call.
```

Comment:
Please remove the first sentence.
It is not helpful to tell what the API does not do, just remove the reference and start with `The root of trush must be created before its first use` or something like that.

Also `If the true random number generator is available in the device it is used ` is also slightly missleading. We TRNG may be used.. but you can also inject the security.
So rewrite the sentence to better indicate that we have two options.